services/spar/src/Spar/Intra/Brig.hs

```diff
@@ -187,20 +135,20 @@ class (Log.MonadLogger m, MonadError SparError m) => MonadSparToBrig m where
 
 createBrigUserSAML ::
   (HasCallStack, MonadSparToBrig m) =>
-  SAML.UserRef ->
+  AuthId ->
   UserId ->
   TeamId ->
   -- | User name
   Name ->
   -- | Who should have control over the user
   ManagedBy ->
   m UserId
-createBrigUserSAML uref (Id buid) teamid uname managedBy = do
+createBrigUserSAML authId (Id buid) teamid uname managedBy = do
```

Comment:
I refactored this so that all public functions are well-specified https://github.com/wireapp/wire-server/pull/1393/commits/c685a5c823971e3d7b91375997cbd277a1322dca